### PR TITLE
[Win10] fix windows store certification errors.

### PR DESCRIPTION
**Pull request by afedchin**

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
Looks ok. Let's see what's left on a new upload.

**Review by MartijnKaijser — APPROVED** [review]

File: cmake/scripts/common/Macros.cmake [hunk 1]
```diff
@@ -667,7 +667,7 @@ macro(core_find_versions)
   include(CMakeParseArguments)
   core_file_read_filtered(version_list ${CORE_SOURCE_DIR}/version.txt)
   core_file_read_filtered(json_version ${CORE_SOURCE_DIR}/xbmc/interfaces/json-rpc/schema/version.txt)
-  string(REPLACE " " ";" version_list "${version_list} ${json_version}")
+  string(REGEX REPLACE "([^ ;]*) ([^;]*)" "\\1;\\2" version_list "${version_list};${json_version}")
```
Comment (wsnipex) [inline cmake/scripts/common/Macros.cmake]: are you saying there are blanks in some versions?

Comment (afedchin) [reply cmake/scripts/common/Macros.cmake]: there are blanks in `COMPANY_NAME` (see fixed version.txt)

**Review by wsnipex — APPROVED** [review]

**Comment by afedchin** [comment]
jenkins build this please

File: tools/buildsteps/windows/BuildSetup.bat [hunk 2]
```diff
@@ -13,7 +13,7 @@ SET sed_exe=%msys_dir%\usr\bin\sed.exe
 
 REM read the version values from version.txt
 FOR /f %%i IN ('%awk_exe% "/APP_NAME/ {print $2}" %base_dir%\version.txt') DO SET APP_NAME=%%i
-FOR /f %%i IN ('%awk_exe% "/COMPANY_NAME/ {print $2}" %base_dir%\version.txt') DO SET COMPANY_NAME=%%i
+FOR /f %%i IN ('%awk_exe% "/COMPANY_NAME/ {$1=""; print $0}" %base_dir%\version.txt') DO SET COMPANY_NAME=%%i
```
Comment (Rechi) [inline tools/buildsteps/windows/BuildSetup.bat]:
```awk: cmd. line:1: /COMPANY_NAME/ {$1="; print $0}
awk: cmd. line:1:                    ^ unterminated string```

Comment (afedchin) [reply tools/buildsteps/windows/BuildSetup.bat]: @Paxxi can you please take a look? I have no idea how it should be

**Comment by afedchin** [comment]
jenkins build this please

File: tools/buildsteps/windows/BuildSetup.bat (hunk 2 above)
Comment (Rechi) [reply tools/buildsteps/windows/BuildSetup.bat]: @MartijnKaijser this is now broken

**Pull request merged**